### ISSUE TITLE
Poor Result for Different Input Channel 

### ISSUE THREAD

**Issue by khoapip** [okankop/Efficient-3DCNNs#34]
Hi! Thank you so much for the awesome paper and source code. I was trying to apply resnext3d on my custom datasets. 

I tested out with inputting RGB images on the pretrain and the output is quite nice (prec@1 87-88 2 classes). However, I tried to tested out with different input channel by saving the npy file of the optical flow (2 channel) and the optical flow stacked with rgb (5 channel), all normalized to 0-255. I changed the input channel for Resnext to be 2 and 5 respectively and the performance is significantly worse than when inputted with a 3 channel (no pretrained) stopping at only about prec@1 60 after around 25-30 epochs. I wonder if there's further adjustment needed to make on the model architecture. Right now I just changed the input channel of the first 3d conv to the input data channel and nothing else. Is there anything I can change to make it better? Thank you a lot for your time!

**Comment by okankop** [comment]
Hi @khoapip, from your description, nothing seems wrong. You are trying to apply data level fusion as I have used once in http://openaccess.thecvf.com/content_cvpr_2018_workshops/papers/w41/Kopuklu_Motion_Fused_Frames_CVPR_2018_paper.pdf  In practice, you should expect it to increase accuracy.

I guess you are training on your dataset that has 2 classes. What would be possible wrong is that your dataset is really small and if you finetune the model with large learning rate, you can hurt the learnt convolutional kernels. I suggest you to apply learning rate of 0.001-0.0001 for finetuning.

**Comment by khoapip** [comment]
Thanks a lot! I copy the weight of the 3 input channels to make it a 5 channel input weights from the fine-tune resnext model and now the performance is much better.
